junit-platform-console-standalone/junit-platform-console-standalone.gradle.kts

```diff
@@ -26,6 +26,8 @@ tasks {
 		}
 	}
 	shadowJar {
+		// https://github.com/junit-team/junit5/issues/2557
+		exclude("module-info.class")
```

Comment:
Exclude all `module-info.class` files - also those potentially in `META-INF/versions/*`.

Reply:
```suggestion
		exclude("**/module-info.class")
```